### ISSUE TITLE
[BUG] Export-PnPFlow unable to export to zip

### ISSUE THREAD

**Issue by LeonArmston** [pnp/powershell#154]
### Reporting an Issue or Missing Feature
I'm trying to Export a Flow to a zip file using Export-PnPFlow and I'm getting The client certificate 'DF4D2C02B7F733BB44849D827A5BC6AFA11B316C' is not allowed (even though I am connecting with user credentials) and it is not allowing me to export the flow to a zip file. Exporting to JSON works fine.
The certificate thumbprint is not on my machine.
This happens every time on every flow (different) and is not intermittent.

Looks like the exact same thing is happening in cli for Microsoft 365 and has the exact same Certificate thumbprint - so might be worth waiting to see if how it is resolved there or if I'm doing anything wrong. 
https://github.com/pnp/cli-microsoft365/issues/2091

My guess is it's a Microsoft Graph issue and not related to PnP.PowerShell?

### Expected behavior 
Flow is exported to a zip file.

### Actual behavior 

![image](https://user-images.githubusercontent.com/12968962/105611066-1818c780-5dab-11eb-8036-12e32a3303cf.png)
![image](https://user-images.githubusercontent.com/12968962/105611078-236bf300-5dab-11eb-973f-c3817c0e8b70.png)

```
PS C:\temp\1> Export-PnPFlow -Environment $FlowEnv -Identity $flow.Name -PackageDisplayName $flow.Properties.DisplayName -AsZipPackage -OutPath "$exportPath.zip" -Force
Export-PnPFlow: {"error":{"code":"AuthorizationFailed","message":"The client certificate 'DF4D2C02B7F733BB44849D827A5BC6AFA11B316C' is not allowed."}}
PS C:\temp\1> Get-PnPException

Message          : {"error":{"code":"AuthorizationFailed","message":"The client certificate 'DF4D2C02B7F733BB44849D827A5BC6AFA11B316C' is not allowed."}}
Stacktrace       :    at PnP.PowerShell.Commands.Utilities.REST.RestHelper.SendMessageAsync(HttpClient httpClient, HttpRequestMessage message) in
                   D:\a\powershell\powershell\src\Commands\Utilities\REST\RestHelper.cs:line 466
                      at PnP.PowerShell.Commands.Utilities.REST.RestHelper.PostAsync(HttpClient httpClient, String url, String accessToken, Object payload, String accept) in
                   D:\a\powershell\powershell\src\Commands\Utilities\REST\RestHelper.cs:line 189
                      at PnP.PowerShell.Commands.Utilities.REST.RestHelper.PostAsync[T](HttpClient httpClient, String url, String accessToken, Object payload, Boolean camlCasePolicy) in
                   D:\a\powershell\powershell\src\Commands\Utilities\REST\RestHelper.cs:line 211
                      at PnP.PowerShell.Commands.Graph.ExportFlow.ExecuteCmdlet() in D:\a\powershell\powershell\src\Commands\Flow\ExportFlow.cs:line 181
                      at PnP.PowerShell.Commands.Base.BasePSCmdlet.ProcessRecord() in D:\a\powershell\powershell\src\Commands\Base\BasePSCmdlet.cs:line 67
                      at System.Management.Automation.CommandProcessor.ProcessRecord()
ScriptLineNumber : 1
```

### Steps to reproduce behavior

```
$cred = Get-PnPStoredCredential -Name LeonDev
Connect-PnPOnline -url "https://tenantname.sharepoint.com/sites/ThePerspective" -Credentials $cred

Export-PnPFlow -Environment "Default-d2af3166-e76b-4d03-9942-fbc79d6b69f9"-Identity ab19c7e5-da28-4a40-80dd-15581e899862 -PackageDisplayName "ExportedFlow" -AsZipPackage -OutPath "ExportedFlow.zip" -Force
```

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)
 PnP.PowerShell 1.2.2 

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

Thanks

**Comment by erwinvanhunen** [comment]
My initial guess is that this is happening between the API endpoint and the actual PowerAutomate environment as this error occurs also when using non-certificate based authentication. As this error occurs also on the M365 CLI and we both use the same endpoint to retrieve flow information.

**Comment by LeonArmston** [comment]
Seems Waldek has now fixed this in M365 CLI with some changes to the code for the export flow command to reflect changes in the underlying API.

Link to code changes: https://github.com/waldekmastykarz/cli-microsoft365/commit/f602924a7f14b4942e00e86a42c43280ec192fc9

**Comment by erwinvanhunen** [comment]
We made the changes and it will be available in the 1.2.7-nightly release.

**Comment by LeonArmston** [comment]
> We made the changes and it will be available in the 1.2.7-nightly release.

Excellent - many thanks!
